src/main/java/com/cursor/demo/DemoApplication.java

```diff
@@ -0,0 +1,20 @@
+package com.cursor.demo;
+
+import com.cursor.demo.controller.Controller;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.ConfigurableApplicationContext;
+
+@SpringBootApplication
+public class DemoApplication {
+
+    public static void main(String[] args) {
+        ConfigurableApplicationContext run = SpringApplication.run(DemoApplication.class, args);
+        program(run);
+    }
+
+    static void program(ConfigurableApplicationContext run) {
```

Comment:
reall? name of the methos is 'program'? change it, please

Reply:
> reall? name of the methos is 'program'? change it, please

done